### PR TITLE
REST API: Add POST /orders/order/{identifier}/shipments

### PR DESCRIPTION
**Pull request by adriendupuis**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | [IBX-5752](https://issues.ibexa.co/browse/IBX-5752), [IBX-5751](https://issues.ibexa.co/browse/IBX-5751)
| Versions      | 4.6
| Edition       | Commerce

Follows #2053

- Add POST /orders/order/{identifier}/shipments (ibexa/shipping#40)
- Fix GET /orders/order/{identifier}/shipments (ibexa/shipping#29)

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/api/rest_api/rest_api_reference/input/examples/orders/POST/ShipmentCreate.json.example [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "ShipmentCreate": {
+    "identifier": "abc",
+    "ShippingMethod": {
+      "_media-type": "application/vnd.ibexa.api.ShippingMethod+json",
+      "_href": "/api/ibexa/v2/shipping/methods/inpost_parcel_locker"
```
Comment (alongosz) [inline docs/api/rest_api/rest_api_reference/input/examples/orders/POST/ShipmentCreate.json.example:6]:
```suggestion
      "_href": "/api/ibexa/v2/shipping/methods/parcel_locker"
```
(for reasons mentioned in some other PR earlier)

File: docs/api/rest_api/rest_api_reference/input/ez-orders.raml [hunk 2]
```diff
@@ -79,30 +79,52 @@ displayName: Orders
...
+                            application/vnd.ibexa.api.Shipment+xml
+                            application/vnd.ibexa.api.Shipment+json
+                    Content-Type:
+                        description: The shipments schema encoded in XML or JSON format.
+                        example: |
+                            application/vnd.ibexa.api.ShipmentCreate+xml
+                            application/vnd.ibexa.api.ShipmentCreate+json
+                body:
+                    application/vnd.ibexa.api.ShipmentCreate+xml:
+                        type: ShipmentCreate
```
Comment (alongosz) [inline docs/api/rest_api/rest_api_reference/input/ez-orders.raml:118]: We probably need XML payload example as well, as we provide for the other endpoints.

File: docs/api/rest_api/rest_api_reference/input/examples/orders/POST/ShipmentCreate.json.example (hunk 1 above)
Comment (adriendupuis) [reply docs/api/rest_api/rest_api_reference/input/examples/orders/POST/ShipmentCreate.json.example]: Done in ef48c4ef4c887959331499dc4263621de072fe85

File: docs/api/rest_api/rest_api_reference/input/ez-orders.raml (hunk 2 above)
Comment (adriendupuis) [reply docs/api/rest_api/rest_api_reference/input/ez-orders.raml:118]: Added in ef48c4ef4c887959331499dc4263621de072fe85 thanks to @alongosz

**Review by alongosz — APPROVED** [review]

**Pull request merged**